src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -376,6 +392,11 @@ internal XmlFormatCollectionReaderDelegate XmlFormatReaderDelegate
                     {
                         if (_helper.XmlFormatReaderDelegate == null)
                         {
+                            if (IsReadOnlyContract)
+                            {
+                                ThrowInvalidDataContractException(_helper.DeserializationExceptionMessage, null /*type*/);
```

Comment:
This can be written as `ThrowInvalidDataContractException(_helper.DeserializationExceptionMessage, type: null);`

Reply:
I'll fix it. This was a literal copy/paste from Framework.